<?xml version="1.0"?>
<launch>
    <arg name="input_pc_topic" default="cloud_downsampled"/>
    <arg name="distance_threshold" default="0.005"/>
    <node pkg="cloud_practice" type="cloud_loader" name="cloud_loader" />
    <node pkg="cloud_practice" type="cloud_filter" name="cloud_filter" />
    <node pkg="cloud_practice" type="cloud_downsampler" name="cloud_downsampler" /> 
    <node pkg="cloud_practice" type="cloud_planar_segmenter" name="cloud_planar_segmenter" >
        <param name="input_pc_topic" value="$(arg input_pc_topic)"/>
        <param name="distance_threshold" value="$(arg distance_threshold)"/>
    </node>
    <node pkg="cloud_practice" type="cloud_clusterer" name="cloud_clusterer" output="screen"/>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find cloud_practice)/rviz/cluster.rviz"/>
</launch>